### PR TITLE
add admin guide for release-monitoring.org

### PR DESCRIPTION
**Pull request by luto**
As discussed in IRC :)

Longer description will follow.

### CODE REVIEW

File: docs/release-monitoring-admin-guide.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+* when removing something, try to create as little impact for others as possible
+* check https://release-monitoring.org/flags once in a while
+* if in doubt, ask in #fedora-apps on freenode IRC
+
+Cases
+=====
+
+Below are a number of common flags submitted by users. Please don't view these as rigid laws, but as guidelines to make our handling of cases consistent. Feel free to adapt them, if it seems reasonable.
+
+Project X is a dupliate!
```
Comment (Zlopez) [inline docs/release-monitoring-admin-guide.rst:20]:
```suggestion
Project X is a duplicate!
```

File: docs/release-monitoring-admin-guide.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+a) does exist: do nothing
+b) doesn't exist: delete the mapping
+c) existed in old, non-EOL version: do nothing
+d) existed in old, EOL version: delete
+
+User's flag is wrong, user misunderstood, ...
+---------------------------------------------
+
+Just close. If you feel very motivated or it seems important, mail them and try to clear it up.
```
Comment (Zlopez) [inline docs/release-monitoring-admin-guide.rst:57]: I never got response to any mail sent, but you could try it.

**Review by Zlopez — COMMENTED** [review]
Two other things:
1) Add the document to index page - I want to have two sections in the future User guide and Admin user guide
2) Add news file - see contribution guidelines

**Comment by luto** [comment]
fixed the comments and rebased onto newest master.

File: news/PR964.dev [hunk 3]
```diff
@@ -0,0 +1 @@
+Add guidelines for admins on release-monitoring.org
```
Comment (Zlopez) [inline news/PR964.dev:1]: Let's rather got with `PR964.other` instead of dev change.

**Review by Zlopez — CHANGES_REQUESTED** [review]
Just one more change and we are ready for merge

File: news/PR964.dev (hunk 3 above)
Comment (luto) [reply news/PR964.dev]: sorry, didn't read closely enough. fixed.

**Review by Zlopez — APPROVED** [review]

**Pull request merged**